src/agent/coverage/src/pe.rs

```diff
@@ -289,6 +280,78 @@ pub fn process_module(
     anyhow::bail!("PE missing codeview pdb debug info")
 }
 
+fn find_pdb_path(pe_path: &Path, cv: &CodeviewPDB70DebugInfo) -> Result<PathBuf> {
+    let cv_filename = CStr::from_bytes_with_nul(cv.filename)?.to_str()?;
+
+    // This field is named `filename`, but it may be an absolute path.
+    // The callee `sym_find_pdb_file_in_path()` handles either.
+    let cv_filename = Path::new(cv_filename);
+
+    // Any nonzero value (cast to a `HANDLE`) should work here.
+    //
+    // The docs warn that we must not intialize the dbghelp symbol handler
+    // with the current process handle when debugging another process. When
+    // we invoke this function, we will often be doing exactly this.
+    //
+    // Assume this is only relevant when making queries about some debuggee's
+    // address space, which we will not do here.
+    let handle = win_util::process::current_process_handle();
+
+    let dbghelp = debugger::dbghelp::lock()?;
+    dbghelp.sym_initialize(handle)?;
```

Comment:
Is this necessary? I can see this if we want to build the cache when not debugging, but I think this is only called in a debugging context where symbols are already loaded using the target process handle.

Note that the value passed can be anything, it need not be a process handle, and in fact, it would probably be better to use something else because the warning in the docs getting unexpected results.

Reply:
So, at one point I actually threaded an optional `handle` parameter through so we could reuse an existing handle-addressed symbol resolver context, and only fell back on this. But, it ended up touching a lot of places, so I shelved it, thinking that it'd be better to revisit more holistically.

After we address everything else, I'll push a commit for you to take a peek at.

Reply:
Take a look at: https://github.com/microsoft/onefuzz/pull/779/commits/98670cfbfbcf88b672f008796dae5ddae7db885c

As discussed, ideally we'll eventually migrate the handle parameter to something like a `DbghelpContext`.